src/components/MaterialUI/Inputs/Switch.js

```diff
@@ -45,13 +45,13 @@ export const useStyles = makeStyles(theme => ({
 		"&:before": {
 			content: props => `"${props.formattedOnCaption}"`,
 			left: theme.spacing(0.9),
-			right: props => (props.readOnly ? theme.spacing(0.9) : 0),
+			right: props => props.readOnly && theme.spacing(0.9),
```

Comment:
Hm... this may need to be checked... the prior conditional operator was setting it to theme.spacing(0.9) or to 0.  Now, you are setting it to true or theme.spacing(0.9)

Reply:
false or  theme.spacing(0.9)
this readOnly props change broke current view

Reply:
Current issue what Oleksand fix here ![image](https://user-images.githubusercontent.com/6649972/106118180-80292e00-615c-11eb-8aa3-457b244602f8.png)